lib/abao.coffee

```diff
@@ -40,7 +40,7 @@ class Abao
       # Parse tests from RAML
       (raml, callback) ->
         if !config.options.server
-          if 'baseUri' in raml
```

Comment:
```if 'baseUri' in raml``` is compiling in unworking code.
Old compiled javascript was
```javascript
            if (indexOf.call(raml, 'baseUri') >= 0) {
```
as you can check this JS don't pass new tests.
New javascript is 
```javascript
            if (raml.baseUri) {
```
and it's okey with new tests.

If you have an idea why ```if 'baseUri' in raml``` has a bug or have a better idea how to fix it, please do the pull.